connectivity/FEATURE_BLE/source/cordio/source/PalPrivateAddressControllerImpl.cpp

```diff
@@ -106,8 +120,12 @@ ble_error_t PalPrivateAddressController::add_device_to_resolving_list(
 )
 {
     if (is_ll_privacy_supported() == false) {
+        tr_error("Operation not supported by LL: RPA resolution");
         return BLE_ERROR_NOT_IMPLEMENTED;
     }
+
+    tr_info("Add RPA to LL resolving list: peer address=%s, type=%s, peer irk=%s, local irk=%s",
+        to_string(peer_address_type), to_string(peer_identity_address), to_string(peer_irk), to_string(local_irk));
```

Comment:
address and type swapped

```suggestion
        to_string(peer_identity_address), to_string(peer_address_type), to_string(peer_irk), to_string(local_irk));
```